guide/14-deep-learning/Labeling text using Doccano.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<p>Some of the images have been stretched and their aspect ratios are distorted. Please keep the original screenshots with only cropping and no stretching.</p><p> </p><p>There needs to be a conclusion, tying the exported data to the next steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done.